win32ss/user/ntuser/defwnd.c

```diff
@@ -529,6 +529,19 @@ DefWndScreenshot(PWND pWnd)
     UserCloseClipboard();
 }
 
+/* Is Window Snap enabled? */
+BOOL IsWindowSnapEnabled(VOID)
+{
+    WCHAR szValue[2];
+    if (RegReadUserSetting(L"Control Panel\\Desktop", L"WindowArrangementActive",
+                           REG_SZ, szValue, sizeof(szValue)))
+    {
+        szValue[RTL_NUMBER_OF(szValue) - 1] = UNICODE_NULL; /* Avoid buffer overrun */
+        return _wtoi(szValue);
+    }
+    return TRUE;
```

Comment:
So window-snap is "by default" enabled?

Reply:
This is what is implemented here, yes.
Since CORE-16379 (request for option to disable it) is from me, I do hereby state that I am okay with that chosen default. Even if it is not the default in 2k3sp2 (which does not have this feature at all yet).

In case you would request to turn it off by default, I would also be okay with that (I am neutral), but that was not explicitly requested by me. So the chosen default will not make **me** reject this PR.